scripts/nn/util.dml

```diff
@@ -216,3 +216,110 @@ threshold = function(matrix[double] X, double thresh)
   out = X > thresh
 }
 
+top_k_row = function(matrix[double] X, integer r, integer k)
+    return (matrix[double] values, matrix[double] indices) {
+  /*
+   * Computes the top k values (i.e. probabilities) and associated
+   * indices (i.e. classes) in the rth row of the input matrix X
+   *
+   * Inputs:
+   * - X: Inputs, of shape (N D).
+   * - r: Input row number of X to look for
```

Comment:
Really small thing: can you add a period to the end of this short sentence?  Also can you do the same elsewhere?